buildpacks/README.md

```diff
@@ -18,25 +18,25 @@ The Cloud Native Buildpacks website describes v3 buildpacks as:
 kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/buildpacks/buildpacks-v3.yaml
 ```
 
-> **NOTE:** This task is currently only compatible with Tekton **v0.6.0** and above, and CNB Platform API 0.2 (lifecycle v0.6.0 and above). For previous Platform API versions, [see below](#previous-platform-api-versions).
+> **NOTE:** This task is currently only compatible with Tekton **v0.11.0** and above, and CNB Platform API 0.3 (lifecycle v0.7.0 and above). For previous Platform API versions, [see below](#previous-platform-api-versions).
 
 ## Parameters
 
-* **`BUILDER_IMAGE`**: The image on which builds will run (must include v3 lifecycle and compatible buildpacks; _required_)
+* **`BUILDER_IMAGE`**: The image on which builds will run. (must include v3 lifecycle and compatible buildpacks; _required_)
 
-* **`USE_CRED_HELPERS`**: Use Docker credential helpers. Set to `"true"` or
-  `"false"` as string a value. (_default:_ `"false"`)
-
-* **`CACHE`**: The name of the persistent app cache volume (_default:_ an empty
+* **`CACHE`**: The name of the persistent app cache volume. (_default:_ an empty
   directory -- effectively no cache)
 
-* **`USER_ID`**: The user ID of the builder image user, as a string value (_default:_ `"1000"`)
+* **`PLATFORM_DIR`**: The name of the platform directory. (_default:_ an empty
```

Comment:
I wonder if there's a way to explain what the purpose of this directory is. I don't know if this is the right place or if it should link to documentation elsewhere.

FWIW, in our [buildpack api docs](https://buildpacks.io/docs/reference/buildpack-api/) we describe this directory as "a directory containing platform provided configuration, such as environment variables."

Reply:
Ah, good point - I'll update the language a bit and add a link to the docs.

Reply:
Updated with more info. The buildpack api docs don't get into the level of detail that would be required to configure the platform directory, so I added some language here and linked to the spec.